### PR TITLE
Update doc (wav-player/tone-player/audio-io) for removed deprecated APIs

### PR DESCRIPTION
**Pull request by SeungbaeShin**
### Change Description ###

Remove deprecated APIs in Multimedia docs (wav-player/tone-player/audio-io).

### API Changes ###

 - ACR:  http://suprem.sec.samsung.net/jira/browse/ACR-1134

### CODE REVIEW

**Comment by journey2w** [comment]
@annie-samsung 
Please review added and changed english sentences.

File: docs/application/native/guides/multimedia/media-playback.md [hunk 1]
```diff
@@ -893,12 +898,14 @@ To start and stop the WAV player:
 
 To start and stop playing a tone:
 
-1. To start playback, use the `tone_player_start()` function.
+1. To start playback, use the `tone_player_start_new()` function.
+
+   The `tone_type_e` (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__TONE__PLAYER__MODULE.html#gaf12912b2c8f9ffe720518ce797506574) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__TONE__PLAYER__MODULE.html#gaf12912b2c8f9ffe720518ce797506574) applications) enumerators define the available values for the tone type (first parameter).
```
[the remark this answers is not in the document]
Comment (SeungbaeShin) [reply docs/application/native/guides/multimedia/media-playback.md]: done

File: docs/application/native/guides/multimedia/media-playback.md [hunk 2]
```diff
@@ -893,12 +898,14 @@ To start and stop the WAV player:
 
 To start and stop playing a tone:
 
-1. To start playback, use the `tone_player_start()` function.
+1. To start playback, use the `tone_player_start_new()` function.
```
[the remark this answers is not in the document]
Comment (SeungbaeShin) [reply docs/application/native/guides/multimedia/media-playback.md]: done

File: docs/application/native/guides/multimedia/media-playback.md [hunk 3]
```diff
@@ -850,7 +850,9 @@ To insert subtitles to a video file:
 
 To start and stop the WAV player:
 
-1. To start the WAV player, use the `wav_player_start()` function.
+1. To start the WAV player, use the `wav_player_start_new()` function.
```
[the remark this answers is not in the document]
Comment (SeungbaeShin) [reply docs/application/native/guides/multimedia/media-playback.md]: done

File: docs/application/native/guides/multimedia/media-playback.md [hunk 4]
```diff
@@ -850,7 +850,9 @@ To insert subtitles to a video file:
...
+
+   The second parameter should be sound information handle which can created by sound_manager_create_stream_information().
```
[the remark this answers is not in the document]
Comment (SeungbaeShin) [reply docs/application/native/guides/multimedia/media-playback.md]: done

**Comment by divyk** [comment]
Thanks for accepting all the comments.

**Comment by mijin85cho** [comment]
As the reviewer leaved the "thank you" comment, I consider it  as she approved this PR.

**Comment by mijin80lee** [comment]
TD Sync Completed.(for Tizen_5.0_prepare)

And please check the following and correct errors.
docs/application/native/guides/multimedia/media-playback.md
line 85: The separated lines are merged as one line.(The callback)
line 162:: The separated lines are merged as one line.(When using the)

docs/application/native/guides/multimedia/raw-audio.md
line 36: Remove '´'.

**Pull request merged**